controller.go

```diff
@@ -411,6 +414,24 @@ func (c *Controller) syncHandler(key string) error {
 				return err
 			}
 
+			id = res.ID
+		} else if c.infraConfig.Provider == "scaleway" {
+			provisioner, _ := provision.NewScalewayProvisioner(c.infraConfig.GetAccessKey(), c.infraConfig.SecretKey, c.infraConfig.OrganizationID)
+
+			userData := makeUserdata(tunnel.Spec.AuthToken, false)
+
+			res, err := provisioner.Provision(provision.BasicHost{
+				Name:       tunnel.Name,
+				OS:         "ubuntu-bionic",
+				Plan:       "DEV1-S",
```

Comment:
Curious if `XS` is live yet and whether it comes with an IP?

Reply:
It seems that the START1-XS is still returned on the API but no more showed on the console.

```
curl -H 'X-Auth-Token: xxxxxxx' \
     -H 'Content-Type: application/json' \
     https://api.scaleway.com/instance/v1/zones/fr-par-1/products/servers -i
```

Returns: 
```json
[...]
"START1-XS": {
    "monthly_price": 1.99,
    "per_volume_constraint": {
      "l_ssd": {
        "min_size": 1000000000,
        "max_size": 200000000000
      }
    },
    "ncpus": 1,
    "ram": 1073741824,
    "arch": "x86_64",
    "volumes_constraint": {
      "min_size": 25000000000,
      "max_size": 25000000000
    },
    "network": {
      "interfaces": [
        {
          "internal_bandwidth": null,
          "internet_bandwidth": 104857600
        }
      ],
      "sum_internal_bandwidth": null,
      "sum_internet_bandwidth": 104857600,
      "ipv6_support": true
    },
    "hourly_price": 0.004,
    "alt_names": [],
    "capabilities": {
      "placement_groups": true,
      "default_boot_type": "bootscript",
      "block_storage": false,
      "boot_types": ["bootscript", "rescue", "local"],
      "hot_snapshots_local_volume": true
    },
    "baremetal": false,
    "gpu": null
  }
[...]
```

Reply:
Does it work and does it have a public IP? If so, then we could use it and save money?